README.md

```diff
@@ -170,31 +170,59 @@ For `sycl_implementation` this can be one of:
 
 You can also specify the additional optional options:
 
+```
 -DSYCL_ACADEMY_INSTALL_ROOT=<path_to_sycl_impl_install_root>
+```
 
 For `<path_to_sycl_impl_install_root>` we recommend you specify the path to the
 root directory of your SYCL implementation installation, though this may not
 always be required.
 
+```
 -DSYCL_ACADEMY_ENABLE_SOLUTIONS=ON
+```
 
 This will enable building the solutions for each exercise as well as the source
 files. This is disabled by default.
 
+```
 -DCMAKE_BUILD_TYPE=Release
+```
 
 The build configuration for all exercises defaults to a debug build if this option is not specified.
 
 #### Additional cmake arguments for DPC++
 
+```
 -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx
+```
 
 This SYCL Academy CMake configuration uses the Intel oneAPI IntelSYCL CMake module package to assist it in its configuration. These command line arguments must be used to initiate this configuration correctly.
 
+`SYCL_TRIPLE` can be used to specify a DPC++ compatible SYCL triple. Possible
```

Comment:
The other options in the README look to be referred to with the `-D` prefix. Could you say `-DSYCL_TRIPLE` here for consistency? Also for the variables in the newly added text below.

Reply:
Thanks, change made.